tools/mafft/main.nf

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env nextflow
+
+// Specify DSL2
+nextflow.enable.dsl=2
+
+// Process definition
+process mafft {
+    tag "${meta.sample_id}"
+
+    publishDir "${params.outdir}/${opts.publish_dir}",
+        mode: "copy",
+        overwrite: true,
+        saveAs: { filename ->
+                      if (opts.publish_results == "none") null
+                      else filename }
+
+    container "quay.io/biocontainers/mafft:7.471--h516909a_0"
+
+    input:
+        val opts
+        tuple val(meta), path (fasta)
+
+    output:
+        tuple val(meta), path ("*.afa"), emit: fasta
```

Comment:
Wrong extension?

Reply:
The file format is fasta, but I use the extensions `'.afa'` or `.mfa` to indicate that it is an aligned fasta file. There is no globally accepted standard file extension for an aligned fasta file. It's worth distinguishing from `.fasta`.